js/iframe-coordinator/src/messages/validationUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { Guard } from 'decoders';
+
+export function createMessageValidator<T>(
+  messageType: string,
+  decoder: Guard<any>
```

Comment:
using any here is a bit of a trick.  We could also use another Type parameter if we really wanted to make it specific.
But I thought:
createMessageValidator<LabledPublication, Publication>

looked a little off, and the end result was the same anyways.